features/environment.py

```diff
@@ -54,6 +54,7 @@ def before_all(context):
     context.emoticons_page = EmoticonsPage(context)
     context.search_page = SearchPage(context)
     context.people_page = PeoplePage(context)
+    context.test_name = "@gtest"
```

Comment:
Do we need dependencies on the test name of "gena" ?

Reply:
As you can see, I writing my name as 'Hena', but you right) I have change this string .